policy/predicate/author.go

```diff
@@ -24,6 +24,41 @@ import (
 	"github.com/palantir/policy-bot/pull"
 )
 
+type OnlyHasAuthorsIn struct {
+	common.Actors `yaml:",inline"`
+}
+
+var _ Predicate = &OnlyHasAuthorsIn{}
+
+func (pred *OnlyHasAuthorsIn) Evaluate(ctx context.Context, prctx pull.Context) (bool, string, error) {
+	author := prctx.Author()
+
+	result, err := pred.IsActor(ctx, prctx, author)
+	if err != nil {
+		return false, "", errors.Wrap(err, "failed to validate author")
+	}
+	if !result {
+		return false, fmt.Sprintf("The pull request author %q does not meet the required membership conditions", author), nil
+	}
+
+	commits, err := prctx.Commits()
+	if err != nil {
+		return false, "", errors.Wrap(err, "failed to get commits")
+	}
+
+	for _, c := range commits {
+		result, err := pred.IsActor(ctx, prctx, c.Author)
```

Comment:
I'd suggest copying and modifying the logic from the existing `HasContribtorIn` predicate, as it has a few advantages:

- It properly considers both the author and the committer of a commit as contributors
- It avoids running `IsActor` on the same user more than once (although the internal caching makes this less important)

Reply:
DONE!